src/dataframerow/utils.jl

```diff
@@ -170,11 +197,37 @@ function group_rows(df::AbstractDataFrame, skipmissing::Bool = false)
 
     # drop group 1 which contains rows with missings in grouping columns
     if skipmissing
-        splice!(starts, 1)
-        splice!(stops, 1)
+        popfirst!(starts)
+        popfirst!(stops)
         ngroups -= 1
     end
 
+    # if row_group_slots can return empty groups, remove them
+    if empty
```

Comment:
is it possible that `empty` is `false`?

Reply:
Woops, yes, that's the case with the fallback method, but for some reason I put `true`...